src/docs/release/breaking-changes/1-22-deprecations.md

```diff
@@ -0,0 +1,768 @@
+* Removed in [#73749][]
+ 
+[`Layer`]: {{site.api}}/flutter/rendering/Layer-class.html
+[`MouseRegion`]: {{site.api}}/flutter/widgets/MouseRegion-class.html
+[`RenderMouseRegion`]: {{site.api}}/flutter/rendering/RenderMouseRegion-class.html
+[`AnnotatedRegionLayer`]: {{site.api}}/flutter/rendering/AnnotatedRegionLayer-class.html
+[`AnnotationResult`]: {{site.api}}/flutter/rendering/AnnotationResult-class.html
+[Breaking Proposal: MouseRegion defaults to opaque; Layers are required to implement findAnnotations]: {{site.github}}/flutter/flutter/issues/38488
+[#37896]: {{site.github}}/flutter/flutter/pull/37896
+[#42953]: {{site.github}}/flutter/flutter/pull/42953
+[#73749]: {{site.github}}/flutter/flutter/pull/73749
+
+---
+
+### `BinaryMessages`
+
+Supported by fix tool: No
+
+The `BinaryMessages` class, its associated static methods and the `defaultBinaryMessenger` getter
+were deprecated and removed. The `defaultBinaryMessenger` instance was moved to `ServicesBinding`.
+This made it possible to register a different default `BinaryMessenger` under testing environment,
+by creating a `ServicesBinding` subclass for testing. Doing so allows you to track the number of
+pending platform messages for synchronization purposes.
+
+ 
+#### Migration Guide
+ 
+Code before migration: | Code after migration:
+-- | --
+`defaultBinaryMessenger` | `ServicesBinding.instance.defaultBinaryMessenger`
+`BinaryMessages` | `BinaryMessenger`
+`BinaryMessages.handlePlatformMessage` | `ServicesBinding.instance.defaultBinaryMessenger.handlePlatformMessage`
+`BinaryMessages.send` | `ServicesBinding.instance.defaultBinaryMessenger.send`
+`BinaryMessages.setMessageHandler` | `ServicesBinding.instance.defaultBinaryMessenger.setMessageHandler`
+`BinaryMessages.setMockMessageHandler` | `ServicesBinding.instance.defaultBinaryMessenger.setMockMessageHandler`
+ 
+#### References
+ 
+API documentation:
+* [`ServicesBinding`][]
+* [`BinaryMessenger`][]
+ 
+Relevant issues:
+* [Flutter synchronization support for Espresso/EarlGrey][]
+
+Relevant PRs:
+* Initially changed in [#37489][]
+* Deprecated in [#38464][]
+* Removed in [#73750][]
+ 
+[`ServicesBinding`]: {{site.api}}/flutter/services/ServicesBinding-mixin.html
+[`BinaryMessenger`]: {{site.api}}/flutter/services/BinaryMessenger-class.html
+[Flutter synchronization support for Espresso/EarlGrey]: {{site.github}}/flutter/flutter/issues/37409
+[#37489]: {{site.github}}/flutter/flutter/pull/37489
+[#38464]: {{site.github}}/flutter/flutter/pull/38464
+[#73750]: {{site.github}}/flutter/flutter/pull/73750
+
+---
+
+### Generic methods for `BuildContext`
+
+Supported by fix tool: Yes
+
+Several methods in `BuildContext` were using `Type` to search for ancestors. Most of those methods
+implied a cast at call site because their return type was a parent type. Moreover the type provided
+was not checked at analysis time even if the type is actually constrained. Making these methods
+generics improves type safety and need less code.
```

Comment:
Nit: "need less code" => "requires less code"

Or maybe these were copied from somewhere and shouldn't change.

Reply:
+1